test/Microsoft.ML.OnnxTransformerTest/OnnxTransformTests.cs

```diff
@@ -416,7 +419,7 @@ public void OnnxModelInMemoryImage()
             // Convert IDataView back to IEnumerable<ImageDataPoint> so that user can inspect the output, column "softmaxout_1", of the ONNX transform.
             // Note that Column "softmaxout_1" would be stored in ImageDataPont.Scores because the added attributed [ColumnName("softmaxout_1")]
             // tells that ImageDataPont.Scores is equivalent to column "softmaxout_1".
-            var transformedDataPoints = ML.Data.CreateEnumerable<ImageDataPoint>(onnx, false).ToList();
```

Comment:
This test needed to be changed because now we're not propagating the input Image column to the output.